charts/loki-distributed/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
-appVersion: 2.2.0
-version: 0.31.1
+appVersion: 2.2.1
+version: 0.32.0
```

Comment:
It's a minor upgrade for Loki and no big changes. You can use a minor incremental for the Helm chart.

Reply:
@Whyeasy Done